data/h3_raster_importer/Makefile

```diff
@@ -12,21 +12,42 @@ data/mapspam/spam2010v2r0_global_prod.geotiff.zip:
 	mkdir -p data/mapspam
 	wget -q -O $@ https://s3.amazonaws.com/mapspam/2010/v2.0/geotiff/spam2010v2r0_global_prod.geotiff.zip
 
+# Download earthstat crop production
+data/earthstat/HarvestedAreaYield175Crops_Geotiff.zip:
+	mkdir -p data/earthstat
+	wget -q -O $@ https://s3.us-east-2.amazonaws.com/earthstatdata/HarvestedAreaYield175Crops_Geotiff.zip
+
+
 # <tiff_folder>: <zipfile>
 # Unzip to folder
 # Only extract the files ending in *_A.tif (all agricultural technology types together)
 data/mapspam/spam2010v2r0_global_prod: data/mapspam/spam2010v2r0_global_prod.geotiff.zip
 	mkdir -p $@
 	unzip -u $< *_A.tif -d $@
 
+
+data/earthstat/earthstat2000_global_prod: data/earthstat/HarvestedAreaYield175Crops_Geotiff.zip
+	mkdir -p $@
+	unzip -j -o -u $< *_Production.tif -d data/earthstat/earthstat2000_global_prod
+	for tiffile in data/earthstat/earthstat2000_global_prod/*.tif; do \
+		table_name=`basename -s .tif "$$tiffile" | tr -d ' \t\n\r' | tr [:upper:] [:lower:]`; \
+		output_file="data/earthstat/earthstat2000_global_prod/$${table_name}_noData.tif";\
+		gdal_translate -a_nodata -9999 -of GTiff "$${tiffile}" "$${output_file}";\
```

Comment:
What's the reason why you're setting the no-data value to -9999? Is it already encoded as -9999 in the GTiff, but not marked as no-data?

Reply:
en mapspam the no data is marked as -1 and in earthstat the no data is marked as NA which means that we are ending up with a NA in an hexagon cell and I'm having issues for importing the table into the database.  I set that no data to -9999 randomly, and for testing that the import worked by setting any value to the no data, but its true that I can change it to -1 to have a similar config as mapspam. What do you think?

Reply:
Actually it looks like EarthStat just has zeros, no nodata areas. Unless I have a different version?

<img width="1226" alt="Screen Shot 2021-09-07 at 9 36 42 AM" src="https://user-images.githubusercontent.com/2185984/132363480-09b8fb62-0f25-4632-8288-60e8b6c6c1da.png">

Reply:
ok, you are right. I was getting confused with the qgis info. I have set the no data in earthstat as 0, if that's ok?

Reply:
Yes I think that's good!